test-case/check-sof-logger.sh

```diff
@@ -256,8 +256,31 @@ main()
         }
     done
 
+    local platf; platf=$(sof-dump-status.py -p)
+    local ERROR_scan=true
+    local OK=true
+
+    case "$platf" in
+        byt|bdw)
+            dlogw 'not looking for ERROR on BYT/BDW because of known DMA issues #4333 and others'
+            ERROR_scan=false
+            ;;
+    esac
+
+    if $ERROR_scan; then
+    for f in "${stdout_files[@]}"; do
```

Comment:
Why not indent?

Reply:
It was supposed to be temporary, I can indent.

Reply:
You pushed me to make more changes, thanks.